File: ProblemSolving/101_4.py
# Lesson learned:
### * split for a single space
### * raw_input in python 2 works for taking input in a while loop, input() doesnt function proper\
### and throws EOF error

# Enter your code here. Read input from STDIN. Print output to STDOUT
pb = {}
n = raw_input()
for i in range(int(n)):
    io = raw_input().strip().split(" ")
    pb[io[0]] = io[1]
while(1): 
    name = raw_input()
    if name in pb:
        print name+"="+pb[name]
    else:
        print "Not found"

File: Go Lang/101_4.go
// 101 Conditional Flow
package main
import "fmt"

func main() {
 //Enter your code here. Read input from STDIN. Print output to STDOUT
    var input int;
    fmt.Scanf("%d",&input);
    //fmt.Println(input);
    if input % 2 == 0 {
        if input >= 2 && input <=5 {
            fmt.Println("Not Weird");
        } else if input > 5 && input < 21 {
            fmt.Println("Weird");
        } else {
            fmt.Println("Not Weird");
        }
    } else {
        fmt.Println("Weird");
    }
}
